pkg/agent/datapath/multiBridgeDatapath.go

```diff
@@ -599,9 +625,15 @@ func InitializeVDS(datapathManager *DpManager, vdsID string, ovsbrName string, s
 	go datapathManager.BridgeChainMap[vdsID][LOCAL_BRIDGE_KEYWORD].(*LocalBridge).cleanLocalEndpointIPAddrWorker(
 		IPAddressCacheUpdateInterval, IPAddressTimeout, stopChan)
 
-	if err := SetPortNoFlood(datapathManager.BridgeChainMap[vdsID][LOCAL_BRIDGE_KEYWORD].(*LocalBridge).name,
-		int(datapathManager.BridgeChainPortMap[ovsbrName][LocalToPolicySuffix])); err != nil {
-		log.Fatalf("Failed to set local to policy port with no flood port mode, %v", err)
+	for _, portSuffix := range []string{LocalToPolicySuffix, LocalToNatSuffix} {
+		if datapathManager.BridgeChainPortMap[ovsbrName][portSuffix] == 0 {
+			log.Infof("Port %s in local bridge doesn't exist, skip set no flood port mode", portSuffix)
+			continue
+		}
+		if err := SetPortNoFlood(datapathManager.BridgeChainMap[vdsID][LOCAL_BRIDGE_KEYWORD].(*LocalBridge).name,
```

Comment:
Are there specific unknown unicast packets that need to be flooded to the nat bridge？

Reply:
no, only specific packets need to output to nat bridge. And the l3forward in nat bridge will depend to the flow from agent

Reply:
local bridge need to add replay logic for flow to nat bridge

Reply:
We need to replay service traffic redirect flow in local bridge at least